controllers/Register.py

```diff
@@ -0,0 +1,60 @@
+import bcrypt
+from db import db
+from flask import Flask,request,jsonify,Blueprint
+from flask_sqlalchemy import SQLAlchemy
+from Models.Users import Users
+from sqlalchemy.exc import IntegrityError
+
+
+app_register = Blueprint('app_register',__name__)
+
+@app_register.route("/register", methods=["POST"])
+def register():
+
+    response = {}
+
+    try:
+        #username_ = request.form['username']
+        #password_ = request.form['password']
+        #password_ = request.form['email']
+
+        username_ = request.json.get('username', None)
+        password_ = request.json.get('password', None)
+        email_ = request.json.get('email', None)
+
+        if not username_:
+            response['message'] = 'Missing username'
```

Comment:
Have a look at the marshmallow package in controllers/MatchUsers.py. It already checks for missing request params in an elegant way!